cli/test/smokehouse/lighthouse-runners/cli.js

```diff
@@ -61,6 +61,8 @@ async function internalRun(url, tmpPath, configJson, options) {
     `${url}`,
     `--output-path=${outputPath}`,
     '--output=json',
+    // native headless, as full headless has several behavior differences that break tests. http://go/rooxt
+    '--chrome-flags="--headless=chrome"',
```

Comment:
Looking at my (stale) branch doing the same, here are some things to consider:

- there is also `lighthouse-runners/bundle.js` to update
- headless-chrome.md could mention this new mode. what I had:

> +There is also the new `--headless=chrome` option, which includes functionality that
> +was explicitly omitted from the original headless browser.
> +
>  ## CLI (xvfb)

Reply:
SG. also i wanted to experiment with using this in CI and getting rid of xvfb, which should (in theory) work.

but i'll do this part in a followup as i expect it'll be slightly more involved

ill address your two bullets in this one